astropy_helpers/sphinx/ext/doctest.py

```diff
@@ -11,7 +11,7 @@
 """
 import re
 from docutils.nodes import literal_block
-from sphinx.util.compat import Directive
+from docutils.parsers.rst import Directive
```

Comment:
I think changing this line is safe as it's around at least since 2012 and now being removed from sphinx.